_pages/continuous-travis.md

```diff
@@ -4,73 +4,73 @@ title: Continuous Deployment - cuz less hassle
 permalink: continuous-travis
 ---
 
-# Continuous Deployment with Travis
+# Travisを使用した継続的デプロイ
 
-*Created by Floor Drees, [@floordrees](https://twitter.com/floordrees)* 
+*Created by Floor Drees, [@floordrees](https://twitter.com/floordrees)* / *翻訳者: [@monya_tto](https://twitter.com/monya_tto)*
 
-### What is this Continuous Deployment thing?
+### 継続的デプロイとは何でしょうか？
 
-Continuous deployment is part of the continuous delivery 'movement'. The idea behind continuous delivery is to automate the software delivery process as far as possible. 
+継続的デプロイは継続的デリバリー「ムーブメント」の一環です。継続的デリバリーの背景にある考え方は、ソフトウェアデリバリープロセスをできる限り自動化することです。
 
-With a working continuous deployment chain in place you'll enforce Git deployments (everything must be committed to be tested and everything must be tested to be deployed), making collaboration easier and deployment faster. So you can focus on making your app even more awesome!
+継続的デプロイチェーンが機能すると、Gitにデプロイ（そこではテストするためにはすべてがコミットされていなければならず、デプロイするためにすべてがテストされていなければなりません）を強制します。それにより共同作業しやすくなり、デプロイが速くなります。そうすれば、アプリをさらに素晴らしいものにすることに集中できるでしょう！
 
-There are a few great companies sailing the continuous wave, in this guide we'll set up continuous deployment for our Ruby on Rails app from GitHub to anynines, using [Travis-ci](http://about.travis-ci.org/). 
+数々の先進的な企業がこの波に乗っています。このガイドにおいては [Travis-ci](http://about.travis-ci.org/) を使って、GitHubからanyninesへRuby on Railsアプリの継続的デプロイのセットアップをしていきましょう。
 
-__COACH__: Talk about the benefits of continuous deployment.
+__COACH__: 継続的デプロイの利点について話しましょう。
 
-###Github, Travis CI and anynines
+### Github、Travis CIとanynines
 
-The first thing we need is an app in a Github repository. And we have just that! Next you'll need to make sure you followed the guide on how to deploy your app via anynines until the very last step.
+まずGithubにリポジトリが必要です。私たちには既にそれがありますね！次にanyninesを使ってデプロイする方法のガイドを最後の手順まで終えたことを確認してください。
 
-Then, we need to create a file called `manifest.yml` in the main directory of your app, so we can save some information about the deployment there. In your terminal run:
+それから、アプリのメインディレクトリに`manifest.yml`という名前のファイルを作成していきましょう。そこにデプロイについての情報を保存することができます。ターミナルで以下のコマンドを実行してください。
 
 {% highlight sh %}
 cf push
 {% endhighlight %}
 
-This will trigger a first deployment to anynines. The cf gem will notice that there is no `manifest.yml` and will ask you a standard set of configuration questions such as the desired number and memory size of your app instances, whether and which services to bind to them and most importantly, whether you want to store this information.
-Please answer this question with a 'hell yes' as it will create the desired `manifest.yml` file!
+これはanyninesへの最初のデプロイの引き金になります。cf gemは`manifest.yml`が無いことを確認すると、基本的な設定に関する質問をしてきます。 例えば、あなたのアプリインスタンスで希望する数とメモリサイズ、どのサービスをバインドするかどうかなどについてです。中でもとりわけ重要な質問はこれらの情報を保存するかということです。
```

Comment:
nits:
> This will trigger a first deployment to anynines.

`cf push` とはanyninesにアプリをデプロイするアプリをデプロイする際に打つコマンドで、ガイドに従ってこのコマンドを実行すると初回デプロイがもたらされるという意図かなと思われます。
「引き金」とするよりも前後の繋がりをみて

```
これにより、anyninesへの初回デプロイが開始されます。
```

とちょっと意訳しちゃっても良いかなと思いました👍

> 基本的な設定に関する質問

構成に関する質問になるため「基本的」ではなく「標準的」でも良さそうだなと思いました👍